tests/ci_build/pip_tests/Dockerfile.pip_dependencies

```diff
@@ -0,0 +1,13 @@
+# -*- mode: dockerfile -*-
+# part of the dockerfile to test pip installations
+
+# add repo to install different Python versions
+RUN apt-get update && apt-get install -y software-properties-common
+RUN add-apt-repository ppa:fkrull/deadsnakes && apt-get update
```

Comment:
This repository has a lot of old/new Python versions to test with. This allows me to install multiple versions of Python 3.